extensions/positron-r/src/r-installation.ts

```diff
@@ -20,6 +20,9 @@ export interface RMetadataExtra {
 	/** R's binary path */
 	readonly binpath: string;
 
+	/** R's RScript path */
+	readonly rScriptPath: string;
```

Comment:
```suggestion
	readonly scriptpath: string;
```

I think matching the existing naming scheme in this interface would be preferred